docs/guides/admin/docs/configuration/basic.md

```diff
@@ -96,8 +96,14 @@ For testing, it is totally fine to keep the internal database, but you are highl
 database for productional use. For more information about database configuration, have a look at the [Database
 Configuration](database.md) section.
 
+Step 6: Elasticsearch Configuration
+-----------------------------------
 
-Step 6: Setting the Storage Directory (optional)
+Since version 9.0, Opencast requires an external Elasticsearch instance. It needs to be started before Opencast.
```

Comment:
This will not end up in any pre-9.x documentation anyway:

```suggestion
Opencast requires Elasticsearch.
```

Reply:
Fine with me. I adjusted the file following your suggestion.